### PR TITLE
Change base Docker image to sitespeed.io

### PR DESCRIPTION
**Pull request by krompaco**
See this as a proposal.

I've been testing it for a couple of weeks as I think there would be some benefits to use the `sitespeedio/sitespeed.io` image instead of their webbrowsers image.

1. Same version number as the npm package
2. Opens up possibility to run tests with the internal command used by sitespeedio's image instead of using the script from npm install (requires some IF-statements in the sitespeed-based tests but could improve stability and lean more on the great work from them)

Not a big change. The additions are mainly to avoid conflicts and to use same aliases as their image does.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,31 +43,38 @@ RUN python3.12 get-pip.py
...
-RUN chown --recursive pptruser:pptruser /usr/src/app
+RUN chown --recursive sitespeedio:sitespeedio /usr/src/runner
 
 # Run everything after as non-privileged user.
-USER pptruser
+USER sitespeedio
 
 RUN npm install
 
+WORKDIR /usr/src/runner/node_modules/sitespeed.io/
```
Comment (7h3Rabbit) [inline Dockerfile:67]: @krompaco why is this needed?

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,11 @@
-FROM sitespeedio/webbrowsers:chrome-124.0-firefox-125.0-edge-123.0
+FROM sitespeedio/sitespeed.io:33.6.0
```
Comment (7h3Rabbit) [inline Dockerfile:1]: @krompaco handle_update_docker method in tools/verify_result.py has to be modified also.

File: Dockerfile (hunk 1 above)
Comment (krompaco) [reply Dockerfile:67]: I tried multiple times and got missing file error unless I did a separate install like this. Could be something with cached folders in the new image.

File: Dockerfile (hunk 2 above)
Comment (krompaco) [reply Dockerfile:1]: I've made an attempt... Seems to work: https://github.com/krompaco/webperf_core/actions/runs/9066534196/job/24909642610#step:6:15

**Comment by 7h3Rabbit** [comment]
@krompaco just an update, the reason for the delay is that I want to test this setup in great depth.
so far haven't had time to setup a docker environment but will hopefully happens soon.

**Comment by 7h3Rabbit** [comment]
@krompaco 
This is what I want to test in depth:
- Is it possible to remove the sitespeed specifict npm install?
- Do everything work? (test with all test against all sites at webperf.se)
- Are there any big benefits of changing parent docker image? Some searchresults indicate better hardware support.
- Is anything missing in the Docker image (if so, can we fix it/give instructions for it?)
- Document use of Docker (read: get a better understanding of docker)
- See if -it and -cap-add=SYS_ADMIN can be removed (SYS_ADMIN is dangerous according to a few searchresults)
- See if there are any benefit of the windows specific ps1 files
- wait for #488 so it can be tested at the same time.

If you (or someone else) can help answer above that would be very helpful 😄
